openwisp_controller/connection/templates/admin/connection/batch_command/execute_command.html

```
@@ -0,0 +1,150 @@
+    {% endif %}
+
+    <fieldset class="module aligned">
+      <h2>{% trans "Command" %}</h2>
+      <div class="form-row field-type{% if form.type.errors %} errors{% endif %}">
+        {{ form.type.errors }}
+        <div class="flex-container">
+          {{ form.type.label_tag }}
+          {{ form.type }}
+        </div>
+        {% if form.type.help_text %}
+        <div class="help"{% if form.type.id_for_label %} id="{{ form.type.id_for_label }}_helptext"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)